### PR TITLE
Update the distributed tracing UI page instructions

### PR DESCRIPTION
**Pull request by rhetoric101**

### CODE REVIEW

File: src/content/docs/distributed-tracing/ui-data/understand-use-distributed-tracing-ui.mdx [hunk 1]
```diff
@@ -246,11 +256,34 @@ When you select a trace from the trace list, you see [a map of services](#trace-
...
 
-### Trace map [#trace-map]
+### Entity map [#entity-map]
 
 If a trace involves more than one entity, and we have information about those entities, the trace page includes a map. For more about how that, see [Maps in context](/docs/new-relic-one/use-new-relic-one/ui-data/maps-in-context).
 
-### Span properties [#span-icons]
+### Waterfall view [#waterfall-view]
+
+The waterfall view is below the entity map, and displays a cascade of a trace's spans. This allows you to see see how spans relate to each other in a vertical layout.
```
Comment (nbaenam) [inline src/content/docs/distributed-tracing/ui-data/understand-use-distributed-tracing-ui.mdx:265]: The "see" word here is repeated.

**Review by nbaenam — APPROVED** [review]
It looks good to me! I've just added a comment.

**Comment by rhetoric101** [comment]
@nbaenam thanks for your review!

File: src/content/docs/distributed-tracing/ui-data/understand-use-distributed-tracing-ui.mdx (hunk 1 above)
Comment (rhetoric101) [reply src/content/docs/distributed-tracing/ui-data/understand-use-distributed-tracing-ui.mdx:265]: @nbaenam thanks for catching this!

File: src/images/distributed-tracing-trace-details-page.png
Comment (paperclypse) [inline src/images/distributed-tracing-trace-details-page.png:1]: @rhetoric101 Is there any info in this image that needs to be smudged out?

**Review by paperclypse — APPROVED** [review]
Seems very thorough! I only had one comment about one of your images. If that image is fine, then I think it's ready to go out.

File: src/images/distributed-tracing-trace-details-page.png
Comment (rhetoric101) [reply src/images/distributed-tracing-trace-details-page.png:1]: @paperclypse I just obfuscated the trace ID. The rest looks well obfuscated from an earlier pass.

**Pull request merged**